salt/modules/mac_service.py

```diff
@@ -262,17 +313,24 @@ def start(name, runas=None):
     # Get service information and file path
     service = _get_service(name)
     path = service['file_path']
+    label = service['plist']['Label']
+
+    # Enable the service
+    launchctl('enable', 'system/{0}'.format(label), runas=runas)
 
     # Load the service: will raise an error if it fails
     return launchctl('load', path, runas=runas)
 
 
-def stop(name, runas=None):
+def stop(name, no_disable=False, runas=None):
```

Comment:
@twangboy, this would be better as `disable=True` to avoid the double negative logic.